client/src/components/LoginSession/LoginSession.scss

```diff
@@ -1,12 +1,18 @@
 .login-session {
+    max-height: 100%;
     &.hidden {
         visibility: hidden;
         opacity: 0;
-        transition: visibility 0s 2s, opacity 2s linear;
+        max-height: 0;
+        transition: visibility 0s 1s, opacity 1s linear, max-height 1s linear;
     }
 
     p {
         margin: 0;
+
+        &:last-of-type {
+            padding-bottom: 20px;
```

Comment:
Don't we have CSS variable for this?

Reply:
Only `$modal-inner-padding` which doesn't make a lot of sense.